### ISSUE TITLE
Doesn't work with Node v4

### ISSUE THREAD

**Issue by samuelcole** [jlongster/prettier#27]
> Block-scoped declarations (let, const, function, class) not yet supported outside strict mode

```
~/Code/main:master$ prettier
/Users/scole/.nodenv/versions/4.6.1/lib/node_modules/prettier/bin/prettier:32
let input;
^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)
    at startup (node.js:139:18)
    at node.js:974:3
zsh: exit 1     prettier
$ node --version
v4.6.1
```

**Comment by jlongster** [comment]
Yeah, good point. This should be fixed inhttps://github.com/jlongster/prettier/commit/4abb8ce5449b68dec1d61d9787802f606a757bc1. Just published 0.0.3 so you can try that.

**Comment by samuelcole** [comment]
fixed thanks!
